tests/Struct/FindologicProductTest.php

```diff
@@ -1095,6 +1095,78 @@ public function testEmptyCategoryNameShouldStillExportCategory()
         $this->assertSame($expectedSecondCatUrl, $catUrls[1]);
     }
 
+    public function testCustomerGroupsAreExportedAsUserGroups(): void
+    {
+        $context = Context::createDefaultContext();
+
+        $customerGroupRepo = $this->getContainer()->get('customer_group.repository');
+        $customerGroupRepo->upsert([
+            [
+                'name' => 'Net customer group',
+                'displayGross' => false
+            ],
+            [
+                'name' => 'Gross customer group',
+                'displayGross' => true
+            ]
+        ], $context);
+
+        $customerGroups = $customerGroupRepo->search(new Criteria(), $context);
+        // Manually sort customer group entities for asserting, since otherwise they would be sorted randomly.
+        $customerGroups->sort(function (CustomerGroupEntity $a, CustomerGroupEntity $b) {
+            if ($a->getDisplayGross() && !$b->getDisplayGross()) {
+                return 1;
+            }
+
+            if ($b->getDisplayGross() && !$a->getDisplayGross()) {
+                return -1;
+            }
+
+            return 0;
+        });
+
+        $productEntity = $this->createTestProduct();
+
+        $findologicProductFactory = new FindologicProductFactory();
+        $findologicProduct = $findologicProductFactory->buildInstance(
+            $productEntity,
+            $this->router,
+            $this->getContainer(),
+            $this->buildSalesChannelContext(Defaults::SALES_CHANNEL, 'http://test.at')->getContext(),
+            $this->shopkey,
+            $customerGroups->getElements(),
+            new XMLItem('123')
+        );
+
+        $customerGroupElements = array_values($customerGroups->getElements());
+        $expectedFirstPriceUserGroup = Utils::calculateUserGroupHash($this->shopkey, $customerGroups->first()->getId());
+        $expectedSecondPriceUserGroup = Utils::calculateUserGroupHash(
+            $this->shopkey,
+            $customerGroupElements[1]->getId()
+        );
+        $expectedThirdPriceUserGroup = Utils::calculateUserGroupHash(
+            $this->shopkey,
+            $customerGroupElements[2]->getId()
+        );
+
+        $actualPrices = $findologicProduct->getPrices();
+        // There must be four prices. One for the "Standard Customer Group", one for the "Net Customer Group",
+        // one for the "Gross Customer Group" and lastly, one without Customer Group.
```

Comment:
Can you pls rename the `$expected...` variables to the same as in this comment. It's better than counting the usergroups